TheFourFifths/consus-client
# Feature/add model tff33 story

### Summary

Provide a form for creating a new model.

### Checklist


Finally, is it ready to be reviewed and merged: yes 🍏 

### How to Review

1. Review this PR first(if you're not trevin) https://github.com/TheFourFifths/consus/pull/18
2. Start server on branch: missed-model-data-tff91
3. Start client
4. Navigate to "View all models" -> "Make new model"
5. Ensure a new model can't be created without a name and description(modal needed for error, possibly different PBI depending on when that's finished)
6. Enter some info in the form and click create model
7. Verify you are redirected to the view all models page.
8. Verify your model with appropriate information is there.

### Links

Put links to Jira tickets, and anything else, here.
- [PR for server](https://github.com/TheFourFifths/consus/pull/18)
- [TFF-82](https://msoese.atlassian.net/browse/TFF-82)
- [TFF-81](https://msoese.atlassian.net/browse/TFF-81)
